tests/admin.js

```diff
@@ -283,4 +286,88 @@ step("Verify Event <message> in Audit log for the <user>", async function (strMe
         assert.ok(await text(strMessage, toRightOf(username), toRightOf(patientIdentifierValue)).exists());
     }
     assert.ok(await text(strMessage, toRightOf(username)).exists());
-});
\ No newline at end of file
+});
+
+step("Enter form name as <formName>", async function (formName) {
+    gauge.dataStore.scenarioStore.put("FormName", formName)
+    await write(formName, into(textBox(below("Form Name"))));
+});
+
+step("Edit form name", async function () {
+    var formName = gauge.dataStore.scenarioStore.get("FormName")
+    await click(link(toRightOf(formName)))
+});
+
+step("Create obs group <arg0> <arg1>", async function (obsName, properties) {
+    await dragAndDrop("ObsGroup", $(".form-builder-row"));
+    await click("Select ObsGroup Source")
+    await write(obsName, into(textBox(below("Control Properties"))))
+    await press('Enter')
+    await click("Coronary Artery Disease, Risk factors")
+    for (var row of properties.rows) {
+        if (row.cells[0] === "Url") {
+            await highlight(row.cells[0])
+            await write("http://snomed.info/sct?fhir_vs=ecl/ <80891009", into(textBox(toRightOf(row.cells[0]))))//I didn't had the previous change that is why I'm hardcoding here,will fix it once the code is merged
+        }
+        else {
+
+            await click(checkBox(toRightOf(row.cells[0])));
+        }
+
+    }
+});
+
+step("Click Publish button", async function () {
+    await click("Publish", { waitForNavigation: true, navigationTimeout: process.env.actionTimeout });
+});
+
+step("Click on Manage Forms", async function () {
+    await click("Manage Forms");
+});
+
+step("Open the form created through form builder", async function () {
+    var formName = gauge.dataStore.scenarioStore.get("FormName")
+    await click($("//a[normalize-space()='" + formName + "']"))
+    await click(formName)
+});
+
+step("Click on delete form", async function () {
+    confirm('Are you sure you want to delete this entire form AND schema?', async () => await accept());
+    await click($("//input[@value='Delete Form']"))
+});
+
+step("Validate the report generated for Snomed form builder form Report <arg0>", async function (observationFormFile) {
```

Comment:
arg0 arg1 -> update this args name where ever you have used.

Reply:
updated